File: data/src/main/java/com/balv/imdb/data/model/MovieNetworkObject.java
package com.balv.imdb.data.model;

import com.google.gson.annotations.SerializedName;

public class MovieNetworkObject {
        @SerializedName("Title")
        public String title;
        @SerializedName("Year")
        public String year;
        public String imdbID;
        @SerializedName("Type")
        public String type;
        @SerializedName("Poster")
        public String poster;
}

File: data/src/main/java/com/balv/imdb/data/model/MovieNetworkObject.kt
package com.balv.imdb.data.model

import com.google.gson.annotations.SerializedName

data class MovieNetworkObject(
    @SerializedName("Title")
    val title: String? = null,

    @SerializedName("Year")
    val year: String? = null,

    @SerializedName("imdbID")
    val imdbID: String,

    @SerializedName("Type")
    val type: String? = null,

    @SerializedName("Poster")
    val poster: String? = null
)
